Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml

```
@@ -425,7 +428,7 @@
     clothingPrototype: ClothingHeadHatHoodWinterHOP
 
 - type: entity
-  parent: [ClothingOuterArmorHoS, ClothingOuterWinterCoatToggleable, BaseSecurityCommandContraband]
+  parent: [ClothingOuterArmorHoS, ClothingOuterWinterCoatToggleableNotAttachableBase, BaseSecurityCommandContraband] # Moffstation - Inherit from abstract base winter coat instead of default toggleable one. This avoids inheriting attachable suit storage
```

```suggestion
  parent: [ClothingOuterArmorHoS, ClothingOuterWinterCoatToggleableNotAttachableBase, BaseSecurityCommandContraband] # Moffstation - Inherit from abstract base winter coat, This avoids inheriting attachable suit storage.
```
I think shortening this would be nice since super long comments are kind anoying IMO